vnc_lite.html

```diff
@@ -120,7 +120,7 @@
             // Note that we use location.href instead of location.search
             // because Firefox < 53 has a bug w.r.t location.search
             const re = new RegExp('.*[?&]' + name + '=([^&#]*)'),
-                  match = ''.concat(document.location.href, window.location.hash).match(re);
+                  match = document.location.href.match(re);
```

Comment:
That regex doesn't match `#`. Since vnc_lite.html doesn't use webutil.js, specifying fragment parameters doesn't work. We should remove that part of the comment here.